cli/lib/kontena/cli/nodes/ssh_command.rb

```diff
@@ -3,9 +3,9 @@ class SshCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
 
-    parameter "NODE_ID", "Node id"
+    parameter "[NODE_ID]", "SSH to Grid node, or --any"
```

Comment:
No need for the "or --any" IMHO.

 Also maybe it should be `[NODE_NAME]`? The fact that the id==name is a design choice the end user should not care about and "id" is usually associated with a funny string of random characters or numbers.